packages/string/README.md

```
@@ -0,0 +1,632 @@
+    // Custom cache store
+    cacheStore?: Map<string, string>;
+
+    // Known acronyms to preserve
+    knownAcronyms?: ReadonlyArray<string>;
+
+    // Locale for script-aware case conversion
+    locale?: string;
+}
+```
+
+### String Splitting
```

markdownlint-cli2 (0.17.2)

Multiple headings with the same content
null

(MD024, no-duplicate-heading)